### PR TITLE
Authorisation manager workflow tests

### PR DESCRIPTION
**Pull request by benbaumann95**
### Description of change

A whole lot of changes..

- Mainly file renaming and restructuring
- Remove the 3 synonyms ('repair', 'job', 'work order') we have for "work order" and just use this 
- Use new /workOrders endpoint as /repairs is now deprecated

2nd commit 14765d1 is adding specs for our new `authorisation manager` user

### CODE REVIEW

File: __test__/work-orders/[id]/update.test.js [hunk 1]
```diff
@@ -1,20 +1,20 @@
-import ChooseJobOptionPage from 'src/pages/repairs/jobs/[id]/close-job'
+import UpdateWorkOrderPage from 'src/pages/work-orders/[id]/update'
```
Comment (benbaumann95) [inline __test__/work-orders/[id]/update.test.js:1]: Seem like github has got this file name confused with https://github.com/LBHackney-IT/repairs-hub-frontend/pull/167/files#diff-62687303bdc964bf8d3779b610177fdf979da50a6a3c66a7ef3302e5bb633208L1-R1

File: __test__/work-orders/[id]/choose-option.test.js [hunk 2]
```diff
@@ -1,20 +1,20 @@
-import UpdateJobPage from 'src/pages/repairs/jobs/[id]/update-job'
+import ChooseWorkOrderOption from 'src/pages/work-orders/[id]/close'
```
Comment (benbaumann95) [inline __test__/work-orders/[id]/choose-option.test.js:1]: Seems like github has got this file rename confused with https://github.com/LBHackney-IT/repairs-hub-frontend/pull/167/files#diff-6905a79791a9124df3a62a132b20d970b030c6fd5be92862eea83adef251c268L1-R1

File: src/pages/work-orders/[id]/choose-option.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import ChooseOption from '../../../components/WorkOrders/ChooseOption'
+import { CONTRACTOR_ROLE, CONTRACT_MANAGER_ROLE } from '../../../utils/user'
+
+const ChooseWorkOrderOption = ({ query }) => {
```
Comment (EGiataganas) [inline src/pages/work-orders/[id]/choose-option.js:4]: How about naming it as `WorkOrderChooseOption`.

By doing that means we will start to follow a pattern. Having `WorkOrder` first as the resource and `ChooseOption` as the action.

File: src/pages/work-orders/[id]/close.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
+import CloseWorkOrder from '../../../components/WorkOrders/CloseWorkOrder'
+import { CONTRACTOR_ROLE, CONTRACT_MANAGER_ROLE } from '../../../utils/user'
+
+const CloseWorkOrderPage = ({ query }) => {
```
Comment (EGiataganas) [inline src/pages/work-orders/[id]/close.js:4]: Same as above, maybe `WorkOrderClosePage`.

File: src/pages/work-orders/[id]/choose-option.js (hunk 3 above)
Comment (EGiataganas) [reply src/pages/work-orders/[id]/choose-option.js:4]: * `WorkOrderChooseOptionPage` if we are having the `Page` at the end of all other files

File: src/pages/work-orders/[id]/update.js [hunk 5]
```diff
@@ -0,0 +1,19 @@
+import UpdateWorkOrder from '../../../components/WorkOrders/UpdateWorkOrder'
+import { CONTRACTOR_ROLE, CONTRACT_MANAGER_ROLE } from '../../../utils/user'
+
+const UpdateWorkOrderPage = ({ query }) => {
```
Comment (EGiataganas) [inline src/pages/work-orders/[id]/update.js:4]: Same as above, maybe `WorkOrderUpdatePage`.

**Comment by EGiataganas** [comment]
@benbaumann95 can you please also add specs for authorisation manager role for the index, logout, and search page?

**Review by siapankina — APPROVED** [review]
great job!

**Pull request merged**